docs/how-to-guides/how-to-use-chainhooks-with-stacks.md

```diff
@@ -113,13 +117,25 @@ Get any transaction emitting given print events predicate
 }
 ```
 
+The following example uses `matches_regex` argument:
+
+```json
+{
+    "if_this": {
+        "scope": "print_event",
+        "contract_identifier": "ST1PQHQKV0RJXZFY1DGX8MNSNYVE3VGZJSRTPGZGM.monkey-sip09",
+        "matches_regex": "(?:^|\\W)vault(?:$|\\W)"
+    },
+}
+```
+
 Get any transaction calling a specific method for a given contract **directly**.
 
 > [!Warning]
 > If the observed method is being called by another contract, this predicate won't detect it.
 
 - `contract-identifier` mandatory argument admits:
-  - string type, fully qualifying the contract to observe. Example: `SP000000000000000000002Q6VF78.pox` `method` mandatory argument admits: - string type, used for specifying the method to observe. Example: `stack-stx`.
+  - string type, fully qualifying the contract to observe. Example: `SP000000000000000000002Q6VF78.pox` `method` mandatory argument admits: - string type, used for specifying the method to observe. Example: `stack-stx`
```

Comment:
`method`? 🤔

Reply:
Method is supposed to go in the next line as a new argument. Fixed the formatting. Thanks!